docs/reference/release-notes/8.0.0-alpha1.asciidoc

```diff
@@ -1,44 +1,520 @@
+* Populate data stream fields when `xOpaqueId` not provided {es-pull}62156[#62156]
+
+Infra/REST API::
+* Do not allow spaces within `MediaType's` parameters {es-pull}64650[#64650] (issue: {es-issue}51816[#51816])
+* Fix: added missing skip {es-pull}41492[#41492]
+* Handle incorrect header values {es-pull}64708[#64708] (issues: {es-issue}51816[#51816], {es-issue}64689[#64689])
+* Ignore media ranges when parsing {es-pull}64721[#64721] (issues: {es-issue}51816[#51816], {es-issue}64689[#64689])
+* `RestController` should not consume request content {es-pull}44902[#44902] (issue: {es-issue}37504[#37504])
+
+Infra/Scripting::
+* Change compound assignment structure to support String concatenation {es-pull}61825[#61825]
+* Fixes casting in constant folding {es-pull}61508[#61508]
+* Several minor Painless fixes {es-pull}61594[#61594]
+
+MULTIPLE AREA LABELS::
+* Avoid `StackOverflowError` due to regex alternate paths {es-pull}61259[#61259] (issue: {es-issue}60889[#60889])
+* Fix cat recovery display of bytes fields {es-pull}40379[#40379] (issue: {es-issue}40335[#40335])
+* Fixes to task result index mapping {es-pull}50359[#50359] (issue: {es-issue}50248[#50248])
+
+Machine Learning::
+* Checking if p-tasks metadata is null before updating state (#41091) {es-pull}41124[#41124] (issues: {es-issue}41090[#41090], {es-issue}41091[#41091])
+* Fix typo Korean HANGUL_JAMO {es-pull}74462[#74462]
+* Handle null value of `FieldCapabilitiesResponse` {es-pull}64327[#64327]
+* Relax the test assertion so that the test always passes {es-pull}44133[#44133] (issue: {es-issue}44132[#44132])
+* Remove erroneous licence {es-pull}58732[#58732] (issue: {es-issue}58682[#58682])
+
+Mapping::
+* Remove assertions that mappings have one top-level key {es-pull}58779[#58779] (issue: {es-issue}58521[#58521])
+
+Packaging::
+* Suppress illegal access in plugin install {es-pull}41620[#41620] (issue: {es-issue}41478[#41478])
+
+SQL::
+* Introduce dedicated node for HAVING declaration {es-pull}71279[#71279] (issue: {es-issue}69758[#69758])
+* Make `RestSqlQueryAction` thread-safe {es-pull}69901[#69901]
+
+Search::
+* Check for negative "from" values in search request body {es-pull}54953[#54953] (issue: {es-issue}54897[#54897])
+* Fix `VectorsFeatureSetUsage` serialization in BWC mode {es-pull}55399[#55399] (issue: {es-issue}55378[#55378])
+* Handle total hits equal to `track_total_hits` {es-pull}37907[#37907] (issue: {es-issue}37897[#37897])
+* Improve error msg for CCS request on node without remote cluster role {es-pull}60351[#60351] (issue: {es-issue}59683[#59683])
+
+Snapshot/Restore::
+* Fix GET /_snapshot/_all/_all if there are no repos {es-pull}43558[#43558] (issue: {es-issue}43547[#43547])
+
+Transform::
+* Fix compile error in `DataFrameTransformProgressIT` {es-pull}43099[#43099]
+
+
+
+[[upgrade-8.0.0-alpha1]]
+[float]
+=== Upgrades
+
+NOT CLASSIFIED::
+* Upgrade to Lucene 8.8.2 on 8.0 {es-pull}71587[#71587]
```

Comment:
```suggestion
```

This seems to be superseded by the entry below, and it's just confusing to release note intermediate steps that were never released.